deploy/tests/src/test/resources/config.properties

```diff
@@ -1,9 +1,10 @@
-private.folder.name = /private
+private.folder.path = /deploy/private
+common.folder.path = /tests/_common
 instance.name = lega
+cega.user = legatest
```

Comment:
is making use of this user causing the changes in `users.py`, for the users to be reloaded?
I am asking because I do not fully follow the need to change the `users.py`?

The test user for the other tests is `dummy`, i believe we can make use of the same user, unless there is a need to use a different user.

Reply:
New users.py doesn't allow me to use auto-generated by test-suite users: with it, I can use only dummy. And I want to avoid relying on pre-created test-user: I prefer to run tests against dynamically generated users. That's why I had to modify users.py